signaling/develop/authentication.mdx

```diff
@@ -34,15 +34,17 @@ a <Vg k="SIG" /> token is a dynamic key generated on your app server that is val
 
```

Comment:
Any solution / workaround for the icon display? This is not a single case.
<img width="635" alt="image" src="https://user-images.githubusercontent.com/101544891/192235930-9a0dbc9e-802e-47f3-b8d2-a53d479ac6a9.png">

Reply:
Done